content/docs/06.enterprise/04.scalability/worker-group.md

```diff
@@ -48,6 +48,24 @@ tasks:
 
 If the `workerGroup.key` property is not provided, all tasks and polling triggers are executed on the default worker group. That default worker group doesn't have a dedicated key.
 
+A `workerGroup.key` can also be assigned dynamically using inputs like in the following example where we specify a `STRING` input to name assign the key at execution:
+
+```yaml
+id: worker-group-dynamic
+namespace: company.team
+
+inputs:
+  - id: workerGroup
```

Comment:
```suggestion
  - id: myWorkerGroup
```
nitpick but seems good to show that this can be anything (doesn't have to be the same as property name `workerGroup`)